tests/pipeline/run_migration_test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+set -euo pipefail
+
+TEST_DIR=$(mktemp -d)
+
+trap 'rm --recursive --force "${TEST_DIR}"' EXIT
+
+cd "${TEST_DIR}" || exit 1
+
+cat > main.c <<EOF
+int main() {}
+EOF
+
+clang main.c
+
+mkdir --parents resume-dir/context
+
+cat > resume-dir/context/model.yml <<EOF
+Version: 1
+Configuration:
+  Disassembly:
+   UseATTSyntax: true
+EOF
+
+revng artifact --resume resume-dir decompile-to-single-file a.out
```

Comment:
Use `/dev/null` instead of `a.out`

Reply:
It would complain about input not being a valid object file.